e2e/setup/global-teardown.ts

```
@@ -0,0 +1,15 @@
+import { execSync } from 'child_process'
+
+import { COMPOSE_FILE } from '../helpers/config'
+
+export default async function globalTeardown(): Promise<void> {
+  if (process.env.E2E_SKIP_TEARDOWN === '1') {
+    console.log('[e2e] E2E_SKIP_TEARDOWN=1 — leaving containers up.')
```

ESLint

[error] Unexpected console statement.

(no-console)